src/Login/Register/register.tsx

```diff
@@ -7,27 +7,41 @@ import {
   Link
 } from 'react-router-dom';
 
-export function Register() {
 
-  //const [loginModel, setLoginModel] = useState(initLoginModel);
+export const handleEmail = (txt: string) => {
```

Comment:
Warum packen Sie diese Funktion nicht in eine eigene .ts-Datei? Und der Test dazu heißt dann genauso nur mit .spec.ts hinten dran. Dann würden Sie Business-Logic und Markup (.tsx-files) sauberer trennen.